vms/proposervm/vm_test.go

```diff
@@ -2542,8 +2542,8 @@ func TestTimestampMetrics(t *testing.T) {
 
 	innerBlock := snowmantest.BuildChild(snowmantest.Genesis)
 
-	outerTime := time.Unix(314159, 0)
-	innerTime := time.Unix(142857, 0)
+	outerTime := time.Unix(upgrade.InitiallyActiveTimeUnix+1000, 0)
```

Comment:
It's hard to interpret what `+1000` means, so we could write this as `upgrade.InitiallyActiveTime.Add(1000 * time.Second)` or `upgrade.InitiallyActiveTime.Add(15 * time.Minute)` (since 1k seconds ~= 15 min)